.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   linux-tests:
     strategy:
       matrix:
-        node: [10, 12, 14, 16]
+        node: [10, 12, 14, 16, 18, 20]
```

Comment:
Have we dropped support for 10-14?

Reply:
I believe that the SDKs only need to support Node LTS, so technically yes, but thinking about leaving it in the CI as it doesn't hurt. If it starts causing issues we can be explicit in dropping support for older node versions